docs/user-docs/latest/settings/managing-global-settings.md

```diff
@@ -132,13 +130,21 @@ To create a new API token:
     | Token duration | Choose a token duration: *7 days*, *30 days*, *90 days*, or *Unlimited*. |
     | Token type     | Choose a token type: *Read-only*, *Full access*, or *Custom*.            |
 
-3. (optional) For the *Custom* token type, define specific permissions for your Collection types and Single types by clicking on the *Collection types* or *Single types* tab, then clicking on the content-type name, and using checkboxes to enable or disable permissions.
-    
-    <!-- TODO: update this Figma screenshot with a real-use one once the feature can be tested -->
+3. (optional) For the *Custom* token type, define specific permissions for your API endpoints by clicking on the content-type name and using checkboxes to enable or disable permissions.
+
```

Comment:
Just a note: be careful when using screenshots in procedures, as they often "cut" the flow and may even hide the remaining steps (especially like here when there's only one step right below the screenshot, some users may not notice it).